core/templates/services/html-length.service.ts

```diff
@@ -27,74 +27,102 @@ export class HtmlLengthService {
   constructor(
     private loggerService: LoggerService,
     private sanitizer: DomSanitizer
-  ) {}
+  ) { }
 
-  /**
-    * ['oppia-noninteractive-image', 'oppia-noninteractive-math']
-    * The above tags constitutes for non text nodes.
-    */
   nonTextTags = ['oppia-noninteractive-math', 'oppia-noninteractive-image'];
-
-  /**
-    * ['li','blockquote','em','strong', 'p', 'a']
-    * The above tags serve as the tags for text content. Blockquote,
-    * em, a, strong occur as descendants of p tag. li tags are
-    * descendants of ol/ul tags.
-    */
   textTags = ['p', 'ul', 'ol'];
```

Comment:
Can you check how strong, em, b, i, li, blockquote are treated and if we need to include them in this list?